core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -32,6 +32,14 @@
 #define KOKKOS_IMPL_OPENMPTARGET_HIERARCHICAL_INTEL_GPU
 #endif
 
+// Define a macro for llvm compiler greater than version 15 and on NVIDIA and
+// AMD GPUs. This would be useful in cases where non-OpenMP standard llvm
+// extensions can be used.
+#if defined(KOKKOS_COMPILER_CLANG) && (KOKKOS_COMPILER_CLANG >= 1500) && \
```

Comment:
Is that the right version to guard for?

Reply:
No that has to change to 17 to be absolutely sure of the support.

Reply:
ping